Jam/Assets/Scripts/MenuSelect.cs

```diff
@@ -4,16 +4,27 @@
 
 public class MenuSelect : MonoBehaviour
 {
-    [SerializeField] List<GameObject> menus = new List<GameObject>();
+    [SerializeField] private Transform _canvas;
+
+    void Start()
+    {
+        //default state: only start menu enabled
+        DisableAllMenus();
+        _canvas.GetChild(0).gameObject.SetActive(true);
```

Comment:
The Start Menu is the logical good choice to enable initially. I suggest injecting the Start Menu GameObject into the MenuSelect.cs script and set that active, instead of the `_canvas.GetChild(0)` . This will leave us flexible to structural changes in the MenuSelect scene